drivers/rtc/rtc_handlers.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020 Paratronic
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <syscall_handler.h>
+#include <drivers/rtc.h>
+
+static inline int z_vrfy_rtc_gettime(struct device *dev, struct timespec *tp)
```

Comment:
I think you'll need to copy tp

Reply:
Why I need to copy? However, I could had:
Z_OOPS(Z_SYSCALL_MEMORY_WRITE(tp, sizeof(*tp)));